server/routes/userAPI.js

```diff
@@ -6,8 +6,7 @@ const mongoose = require("mongoose");
 const router = express.Router();
 
 // GET whether id is unique
-router.get("/api/user/is_unique", (req, res) => {
-
+router.get("/api/user/is_unique", auth, (req, res) => {
```

Comment:
Dashes are [preferred](https://developers.google.com/search/docs/advanced/guidelines/url-structure?hl=en&visit_id=637467458802697471-4015835834&rd=1) to underscores in urls